shared/torngit/github.py

```diff
@@ -350,7 +350,7 @@ async def get_repository(self, token=None):
             ),
         )
 
-    async def list_repos_using_installation(self, username):
+    async def list_repos_using_installation(self, username=None):
```

Comment:
`username` is not used so I made this argument optional.  Maybe we should just remove it entirely and make the corresponding change in `worker` as well.